### PR TITLE
fix: support franklin- prefix

### LINKED ISSUE #350: monitoring setup should also support `franklin` prefix
The code at:

https://github.com/adobe/helix-ops/blob/bc2c79314f6e8b6d248203ee2c7b5d853c5633b5/src/monitoringSetup.js#L27

should also support a `franklin-` prefix, or the resulting monitor will have an `undefined` in its URL, as in:
```
https://eynvwoxb7l.execute-api.us-east-1.amazonaws.com/helix-services/undefined/v1/_status_check/healthcheck.json
```

Suggest to change this line to:
```
shortName = name.split('-').slice(1).join('-');
```

### PR DESCRIPTION
**Pull request by rofe**
Fix #350

### CODE REVIEW

File: src/monitoringSetup.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function getServiceInfo() {
   try {
     const json = JSON.parse(fs.readFileSync('package.json'));
     name = json.name;
-    [, shortName] = name.split('helix-');
+    [, shortName] = name.split('-');
```
Comment (dominique-pfister) [inline src/monitoringSetup.js:27]: Hm, that will just return the 2nd segment, e.g.
```
> [,shortName] = 'franklin-query-service'.split('-')
[ 'franklin', 'query', 'service' ]
> shortName
'query'
```
What about:
```
[, shortName] = name.split('-').slice(1).join('-');
```
?

**Review by dominique-pfister — APPROVED** [review]

**Review by tripodsan — APPROVED** [review]

File: test/monitoring/specs/adobeio_only.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
 {
+  "cwd": "franklin-project",
   "parameters": {},
   "output": [
     "setup",
     "--url",
     "https://adobeioruntime.net/api/v1/web/helix/helix-services/monitoring-test@v1/_status_check/healthcheck.json",    "--name",
-    "@adobe/helix-monitoring-test"
+    "@adobe/franklin-monitoring-test"
```
Comment (dominique-pfister) [inline test/monitoring/specs/adobeio_only.json:8]: `cwd` is the franklin-project, so we expect the project name to have a franklin in it.

File: test/testMonitoringSetup.js [hunk 3]
```diff
@@ -69,30 +69,33 @@ describe('Testing monitoring setup', () => {
...
-    shell.cd('-');
   });
 
   const specs = path.resolve(MONITORING, 'specs');
   fse.readdirSync(specs).forEach((filename) => {
     const name = filename.substring(0, filename.length - 5);
-    const { env, parameters, output } = fse.readJSONSync(path.resolve(specs, filename), 'utf8');
+    const {
+      env, parameters, output, cwd = 'helix-project',
+    } = fse.readJSONSync(path.resolve(specs, filename), 'utf8');
```
Comment (dominique-pfister) [inline test/testMonitoringSetup.js:79]: use `helix-project` folder unless told otherwise

File: test/testMonitoringSetup.js [hunk 4]
```diff
@@ -69,30 +69,33 @@ describe('Testing monitoring setup', () => {
...
     it(`Testing ${name}`, async () => {
       setEnv(env);
       const params = applyDefaults(parameters, defaults);
+      params.tool_path = MONITORING;
```
Comment (dominique-pfister) [inline test/testMonitoringSetup.js:83]: the default tool_path is `.` which doesn't get replaced later, so do it here

File: .circleci/config.yml [hunk 5]
```diff
@@ -2,10 +2,7 @@ version: 2.1
 executors:
   node14:
     docker:
-    - image: circleci/node:14
-    environment:
-      NPM_CONFIG_PREFIX: ~/.npm-global
-      LC_ALL: en_US.UTF-8
+    - image: cimg/node:16.15
```
Comment (dominique-pfister) [inline .circleci/config.yml:5]: replaced with node16, NPM_CONFIG_PREFIX no longer needed (and causes an error on startup)

**Review by tripodsan — APPROVED** [review]

**Pull request merged**